js/ftx.js

```diff
@@ -2283,4 +2285,48 @@ module.exports = class ftx extends Exchange {
         const result = this.safeValue (response, 'result', {});
         return this.parseFundingRate (result, market);
     }
+
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetSpotMarginBorrowRates ();
+        const timestamp = this.milliseconds ();
+        const result = this.safeValue (response, 'result');
+        const rates = [];
+        for (let i = 0; i < result.length; i++) {
+            const rate = result[i];
+            const currency = this.safeCurrencyCode (this.safeString (rate, 'coin'));
+            rates[currency] = {
+                'currency': currency,
+                'rate': this.safeNumber (rate, 'previous'),
+                'period': 3600000,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'info': rate,
+            };
+        }
+        return rates;
+    }
+
+    async fetchBorrowRate (code, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetSpotMarginBorrowRates (params);
+        const timestamp = this.milliseconds ();
+        const currencyId = this.currency (code)['id'];
+        const result = this.safeValue (response, 'result');
+        for (let i = 0; i < result.length; i++) {
+            const rate = result[i];
+            const coin = this.safeCurrencyCode (this.safeString (rate, 'coin'));
+            if (coin === currencyId) {
+                return {
+                    'currency': currencyId,
```

Comment:
We should use the unified currency code here.

Reply:
The value is unified, do you mean change the key to `code`?

Reply:
@samgermain sorry, i was confused by the variable naming, cause it says `currencyId`...

In general:

- currency id = currencyId = currency['id'] = non-unified exchange specific id of the currency
- currency code = code= currency['code'] = unified ccxt code of the currency
- currency = a unified currency structure like this: https://github.com/ccxt/ccxt/wiki/Manual#currency-structure

Reply:
@samgermain actually, it is not unified – you're putting an exchange specific currency id into the field that should contain a unified currency code.